config/initializers/avo.rb

```diff
@@ -0,0 +1,109 @@
+# For more information regarding these settings check out our docs https://docs.avohq.io
+Avo.configure do |config|
+  ## == Routing ==
+  config.root_path = "/developers/avo"
```

Comment:
OK, @adrianthedev, let's go with `/admin` as the prefix. I think that's a good suggestion since it's not exclusively a developer concern in the same way that Sidekiq (probably) is.

```suggestion
  config.root_path = "/admin/avo"
```